internal/follower/follower.go

```diff
@@ -84,8 +85,8 @@ var (
 
 // New creates a Follower configured with the passed parameters and ready to be used.
 // It does not check the correctness of the parameters, make sure everything is initialized.
-func New(ctx context.Context, ref string, printer *output.Printer, config *Config) (*Follower, error) {
-	reg, err := utils.GetRegistryFromRef(ref)
+func New(ref string, printer *output.Printer, conf *Config) (*Follower, error) {
```

Comment:
Would avoid this refactor in this PR.

Reply:
Mostly removed the ctx here because linting was complaining since its not used anymore, but I can add it back to the function signature.

Reply:
Sorry, I was referring to `config` parameter which was renamed to `conf`.

Reply:
Ah I see sorry I misunderstood. I changed that one also because of linting since the`config` var shadows the `config` module. Thats imported [here](https://github.com/falcosecurity/falcoctl/blob/09e5b0276631edf8642233228dfd96e54bf529a9/internal/follower/follower.go#L34).

I can change it back to config if you are fine with the shadowing.

Reply:
Oh got it, then let's make the linter happy and keep your changes. Thanks!